### PR TITLE
[O] Reorganize contributor documentation

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+shave](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#lazy-programming).
+
+## Questions
+
+It's always ok to ask questions. Our systems are large, nobody will be
+an expert in all the systems. Once you find the answer, document it in
+the first place you looked. That way, the next person will be brought
+up to speed even quicker.
+
+[!["I try not to make fun of people for admitting they don't know things, because for each thing 'everyone knows' by the time they're adults, every day there are, on average, 10,000 people in the US hearing about it for the first time. If I make fun of people, I train them not to tell me when they have those moments. And I miss out on the fun." "Diet coke and mentos thing? What's that?" "Oh man! We're going ot the grocery store." "Why?" "You're one of today's lucky 10,000." (xkcd, May 2012)](https://imgs.xkcd.com/comics/ten_thousand.png)](https://xkcd.com/1053/https://xkcd.com/1053/)
```
Comment (jonahwilliams) [inline CODE_OF_CONDUCT.md]: s/ot/sto

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,296 +3,73 @@ Contributing to Flutter
...
+documentation, or to help out in any other way.
+
+We grant commit access (which includes full rights to the issue
+database, such as being able to edit labels) to people who have gained
+our trust and demonstrated a commitment to Flutter.
+
+This document focuses on what is needed to contribute by writing code
+and submitting pull requests for the Flutter framework. For
+information on contributing in other ways, see [the CONTRIBUTING page
+on flutter.io](https://flutter.io/CONTRIBUTING).
```
Comment (dnfield) [inline CONTRIBUTING.md]: This is a 404 right now.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Hixie) [reply CODE_OF_CONDUCT.md]: good catch, thanks. Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Hixie) [reply CONTRIBUTING.md]: fixed. This got renamed to https://flutter.io/community recently. Thanks.

**Review by cbracken — APPROVED** [review]

**Comment by Hixie** [comment]
Thanks. Will land on green.

**Pull request merged**